src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -40,6 +40,7 @@
   <ItemGroup>
     <!-- Dependencies needed for Storage Providers -->
     <PackageReference Include="Azure.Core" Version="1.38.0" />
+    <PackageReference Include="Azure.Data.Tables" Version="12.8.3" />
```

Comment:
Should we use the latest version of this package? https://www.nuget.org/packages/Azure.Data.Tables/12.9.0

Reply:
WebHost is currently referencing 12.8.3 so I would like to avoid more changes to this PR whenever possible. Are we OK with bumping versions and normalize dependencies in a future PR?

Reply:
yea, Let's use the same existing version in that case for this PR.